plugins/modules/ovirt_disk.py

```diff
@@ -73,6 +74,7 @@
             - Specify format of the disk.
             - Note that this option isn't idempotent as it's not currently possible to change format of the disk via API.
         choices: ['raw', 'cow']
+        default: 'cow'
```

Comment:
Does it make sense to have a default format?

Reply:
Not sure.
There is already a default format [1], this just updates the documentation.
[1] https://github.com/oVirt/ovirt-ansible-collection/blob/master/plugins/modules/ovirt_disk.py#L654

Reply:
OK, I missed it's already default in the code